src/main/java/racingcar/Car.java

```diff
@@ -0,0 +1,25 @@
+package racingcar;
+
+public class Car {
+    private static final int MOVE_THRESHOLD = 4;
+    private final String name;
+    private int location = 0;

```

Comment:
리뷰 내용에 공백도 의미를 가진다 라고 되어있었어요 ! 별거 아니지만 이렇게 바꿔보는게 어떨까요 

```suggestion
    private static final int MOVE_THRESHOLD = 4;
    
    private final String name;
    private int location = 0;

```

Reply:
옹!! 변수 선언에 공백을 넣어 상수와 일반 변수가 더 잘 구분되어 보이겠네요! 좋은것같습니다!!